pkg/antiaffinity/session.go

```diff
@@ -0,0 +1,95 @@
+/*
```

Comment:
the funcs in this file should move the existing `session.go`

Reply:
I am planning a follow up PR to figure out the session creation that is present in 4 different controllers. Will address it in that PR.